db/migrate/20240517085141_remove_lesson_references.rb

```diff
@@ -0,0 +1,17 @@
+class RemoveLessonReferences < ActiveRecord::Migration[7.0]
```

Comment:
For some reason rolling back this migration fails. To rollback run `rake db:rollback` or `rails db:rollback`